piker/brokers/deribit/api.py

```diff
@@ -95,9 +95,14 @@ class JSONRPCResult(Struct):
     error: Optional[dict] = None
     usIn: int 
     usOut: int 
-    usDiff: int 
+    usDiff: int
     testnet: bool
 
+class JSONRPCChannel(Struct):
```

Comment:
am i missing where this new type is this being used?

if this is the msg that can also be received for non-request results but that show up on the ws channel maybe at least use this in the type annotation below?

See my comment below 😸